index.html

```diff
@@ -46,5 +47,178 @@ <h1>I’m Rashmi.<br> to see<br> you!</h1>
+                            </ul>
+                        </div>
+                        <p>A daily selection of privately<br> 
+                            personalized reads; no accounts or<br> 
+                            sign-ups required.
+                        </p>
+                        <div class="lang">
+                            <ul>
+                                <li>HTML</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                        <div >
+                             <button class="button-A">See project</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        
+        <section class="about-myself">
+            <div class="Myself">
+                <section class="welcome">
+                    <h1>About Myself</h1>
+                </section>
+                <div class="info">
+                    <p>I’m a software developer! I can help you<br> 
+                        build a product , feature or website Look<br> 
+                        through some of my work and<br> 
+                        experience! If you like what you see and<br> 
+                        have a project you need coded, don’t<br> 
+                        hesitate to contact me.
+                    </p>
+                </div>
+                <div class="connect">
+                  <p>LET'S CONNECT</p>
+                </div>
+                <div class="icons">
+                    <ul>
+                        <li><a href="#"><img src="img/Vector.png"></a></li>
+                        <li><a href="#"><img src="img/Linkedin icon (1).png"></a></li>
+                        <li><a href="#"><img src="img/Vector (1).png"></a></li>
+                        <li><a href="#"><img src="img/Vector (3).png"></a></li>
+                        <li><a href="#"><img src="img/Vector (4).png"></a></li>
+                    </ul>
+                </div>
+            </div>  
+            <div>
+                <button class="button-A">Get my resume</button>
+           </div>
+           <br>
+            <div class="grid-container-a">
+                <div class="grid-item-a">
+                    <div class="code-lang">
+                        <img src="img/Ellipse 1.png">
+                        javaScript
+                    </div>
+                </div>
+                <div class="grid-item-a">
+                    <div class="code-lang">
+                        <img src="img/Ellipse 1 (1).png">
+                        HTML</div>
+                    </div>
+                <div class="grid-item-a">
+                    <div class="code-lang">
+                        <img src="img/Ellipse 1 (2).png">
+                        CSS
+                    </div>
+                </div>
+            </div>
```

Comment:
According to the project requirements, I suggest you use the `ul` and `li` tags to list the elements (Languages, Frameworks, and Skills).
<img width="957" alt="Screenshot 2023-06-03 at 6 28 24 PM" src="https://github.com/siriwardhanage/My-first-portfolio-/assets/91237525/45d315cf-91d7-40db-94ec-69eb4baf60e9">